bin/Data/Scripts/Editor/EditorSpawn.as

```diff
@@ -22,6 +22,7 @@ float spawnRadius = 0;
 bool useNormal = true;
 bool alignToAABBBottom = true;
 bool spawnOnSelection = false;
+bool parentToSelection = true;
```

Comment:
I believe this option should be turned off by default, because you have changed the default editor behavior. This must be justified. If you place monsters, stones, objects with rigidbodies and so on on a scene, they do not have to be children of the floor at all, as they can move arbitrarily.

Reply:
Good point.